test/e2e/operator.go

```diff
@@ -226,22 +230,22 @@ var _ = Describe("ARO Operator - Cluster Monitoring ConfigMap", func() {
 			return true, nil
 		}
 
+		By("waiting for the ConfigMap to make sure it exists")
 		err := wait.PollImmediate(30*time.Second, 15*time.Minute, configMapExists)
 		Expect(err).NotTo(HaveOccurred())
 
+		By("deleting for the ConfigMap")
 		err = clients.Kubernetes.CoreV1().ConfigMaps("openshift-monitoring").Delete(context.Background(), "cluster-monitoring-config", metav1.DeleteOptions{})
 		Expect(err).NotTo(HaveOccurred())
 
+		By("waiting for the ConfigMap to make sure it was restored")
 		err = wait.PollImmediate(30*time.Second, 15*time.Minute, configMapExists)
 		Expect(err).NotTo(HaveOccurred())
-
-		_, err = clients.Kubernetes.CoreV1().ConfigMaps("openshift-monitoring").Get(context.Background(), "cluster-monitoring-config", metav1.GetOptions{})
-		Expect(err).NotTo(HaveOccurred())
```

Comment:
This get doesn't make much sense since we are doing exactly the same thing in `PollImmediate` just above. `wait.PollImmediate` will error on timeout. Also we should probably look into replacing `wait.PollImmediate` with ginkgo's `Eventually`.